### PR TITLE
Implement Python API and add unit test for RandomTask

### PR DESCRIPTION
**Pull request by JYPark09**
This revision includes:

- Python API (constants)

- Python API (class)
    - Create `AccountInfo` class.
    - Create `Card` class.
    - Create `Cards` class.
    - Create `DeckCode` class.
    - Create `DeckInfo` class.
    - Create `AccuntLoader` class.
    - Create `PowerLoader` class.
    - Create `TargetingPredicates` class.

- Python API (enums)
    - Create ActionEnums.
    - Create AuraEnums.
    - Create CardEnums.
    - Create GameEnums.
    - Create TargetingEnums.
    - Create TaskEnums.
    - Create TriggerEnums.

- Fix codecov issue
    - Add test for RandomTask when playables count < amount

### CODE REVIEW

File: Tests/PythonTests/test_deck_info.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.show_card_list()
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 1
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 2
+    assert deck.add_card(mage_cards[0].id, 1) == False
+    assert deck.add_card(mage_cards[1].id, 3) == False
+    assert deck.add_card(druid_cards[0].id, 1) == False
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 15
           

Complexity increasing per file
==============================
- Tests/PythonTests/test_deck_code.py  49
- Tests/PythonTests/test_account_info.py  8
- Tests/PythonTests/test_cards.py  18
- Tests/PythonTests/test_card.py  2
- Tests/PythonTests/test_deck_info.py  14
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/RosettaStone/RosettaStone/pullRequest?prid=4593740&bid=15422942)

File: Tests/PythonTests/test_deck_info.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    druid_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.DRUID)
+    mage_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.MAGE)
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.show_card_list()
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 1
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 2
+    assert deck.add_card(mage_cards[0].id, 1) == False
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    assert deck.add_card(mage_cards[1].id, 3) == False
+    assert deck.add_card(druid_cards[0].id, 1) == False
+    deck.show_card_list()
+
+    assert deck.unique_num_of_cards() == 1
+    assert deck.num_of_cards() == 2
+
+    assert deck.card(0)[1] == 2
+    assert deck.delete_card(mage_cards[0].id, 1) == True
+    assert deck.delete_card(mage_cards[0].id, 4) == False
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    mage_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.MAGE)
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.add_card(mage_cards[0].id, 1)
+    
+    pri_deck = deck.primitive_deck()
+    assert pri_deck[0].id == mage_cards[0].id
+
+def test_card_ids():
+    INKEEPER_EXPERT_WARLOCK = 'AAEBAfqUAwAPMJMB3ALVA9AE9wTOBtwGkgeeB/sHsQjCCMQI9ggA'
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [local variable 'INKEEPER_EXPERT_WARLOCK' is assigned to but never used](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_cards.py [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+def test_all_cards():
+	cards = pyRosetta.Cards.all_cards()
+
+	assert len(cards) == 7735
+
+def test_all_standard_cards():
+	cards = pyRosetta.Cards.all_standard_cards()
+
+	for card in cards:
+		assert card.is_standard_set() == True
```
Comment (utilForever) [inline Tests/PythonTests/test_cards.py]: Codacy Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    assert deck.add_card(mage_cards[0].id, 1) == False
+    assert deck.add_card(mage_cards[1].id, 3) == False
+    assert deck.add_card(druid_cards[0].id, 1) == False
+    deck.show_card_list()
+
+    assert deck.unique_num_of_cards() == 1
+    assert deck.num_of_cards() == 2
+
+    assert deck.card(0)[1] == 2
+    assert deck.delete_card(mage_cards[0].id, 1) == True
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    mage_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.MAGE)
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.show_card_list()
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 1
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 2
+    assert deck.add_card(mage_cards[0].id, 1) == False
+    assert deck.add_card(mage_cards[1].id, 3) == False
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    assert deck.add_card(druid_cards[0].id, 1) == False
+    deck.show_card_list()
+
+    assert deck.unique_num_of_cards() == 1
+    assert deck.num_of_cards() == 2
+
+    assert deck.card(0)[1] == 2
+    assert deck.delete_card(mage_cards[0].id, 1) == True
+    assert deck.delete_card(mage_cards[0].id, 4) == False
+    assert deck.delete_card(druid_cards[0].id, 1) == False
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+def test_card_ids():
+    INKEEPER_EXPERT_WARLOCK = 'AAEBAfqUAwAPMJMB3ALVA9AE9wTOBtwGkgeeB/sHsQjCCMQI9ggA'
+    deck = pyRosetta.DeckCode.decode('AAEBAfqUAwAPMJMB3ALVA9AE9wTOBtwGkgeeB/sHsQjCCMQI9ggA').card_ids()
+    
+    assert len(deck) == 30
+
+    for card_id in deck:
+        card = pyRosetta.Cards.find_card_by_id(card_id)
+
+        assert (card.card_class() == pyRosetta.CardClass.WARLOCK or card.card_class() == pyRosetta.CardClass.NEUTRAL) == True
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_account_info.py [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+def test_deck_control():
+    player = pyRosetta.AccountInfo()
+
+    player.show_deck_list()
+    assert player.create_deck('deck1', pyRosetta.CardClass.INVALID) == False
+
+    player.create_deck('deck2', pyRosetta.CardClass.DREAM)
+    player.create_deck('deck3', pyRosetta.CardClass.DRUID)
+
```
Comment (utilForever) [inline Tests/PythonTests/test_account_info.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_cards.py [hunk 11]
```diff
@@ -0,0 +1,216 @@
...
+
+def test_all_wild_cards():
+	cards = pyRosetta.Cards.all_wild_cards()
+
+	for card in cards:
+		assert card.is_wild_set() == True
```
Comment (utilForever) [inline Tests/PythonTests/test_cards.py]: Codacy Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.add_card(mage_cards[0].id, 1)
+    
+    pri_deck = deck.primitive_deck()
+    assert pri_deck[0].id == mage_cards[0].id
+
+def test_card_ids():
+    INKEEPER_EXPERT_WARLOCK = 'AAEBAfqUAwAPMJMB3ALVA9AE9wTOBtwGkgeeB/sHsQjCCMQI9ggA'
+    deck = pyRosetta.DeckCode.decode('AAEBAfqUAwAPMJMB3ALVA9AE9wTOBtwGkgeeB/sHsQjCCMQI9ggA').card_ids()
+
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_account_info.py [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+    assert player3.email() == '2'
+    assert player3.nickname() == 'name2'
+    assert player3.num_of_deck() == 0
+
+def test_deck_control():
+    player = pyRosetta.AccountInfo()
+
+    player.show_deck_list()
+    assert player.create_deck('deck1', pyRosetta.CardClass.INVALID) == False
```
Comment (utilForever) [inline Tests/PythonTests/test_account_info.py]: Codacy Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+    assert deck2.deck_class() == pyRosetta.CardClass.MAGE
+    assert deck2.num_of_cards() == 0
+
+def test_card_control():
+    druid_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.DRUID)
+    mage_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.MAGE)
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.show_card_list()
+    assert deck.add_card(mage_cards[0].id, 1) == True
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py (hunk 4 above)
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [Unused variable 'INKEEPER_EXPERT_WARLOCK'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+    mage_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.MAGE)
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.add_card(mage_cards[0].id, 1)
+    
+    pri_deck = deck.primitive_deck()
+    assert pri_deck[0].id == mage_cards[0].id
+
+def test_card_ids():
+    INKEEPER_EXPERT_WARLOCK = 'AAEBAfqUAwAPMJMB3ALVA9AE9wTOBtwGkgeeB/sHsQjCCMQI9ggA'
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [local variable 'INKEEPER_EXPERT_WARLOCK' is assigned to but never used](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [Unused variable 'INKEEPER_EXPERT_WARLOCK'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- Tests/PythonTests/test_card.py  2
- Tests/PythonTests/test_account_info.py  8
- Tests/PythonTests/test_cards.py  18
- Tests/PythonTests/test_deck_code.py  49
- Tests/PythonTests/test_deck_info.py  14
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/RosettaStone/RosettaStone/pullRequest?prid=4593740&bid=15422942)

File: Tests/PythonTests/test_account_info.py [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+
+def test_deck_control():
+    player = pyRosetta.AccountInfo()
+
+    player.show_deck_list()
+    assert player.create_deck('deck1', pyRosetta.CardClass.INVALID) is False
+
+    player.create_deck('deck2', pyRosetta.CardClass.DREAM)
+    player.create_deck('deck3', pyRosetta.CardClass.DRUID)
+
```
Comment (utilForever) [inline Tests/PythonTests/test_account_info.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+    deck = pyRosetta.DeckCode.decode('AAEBAfqUAwAPMJMB3ALVA9AE9wTOBtwGkgeeB/sHsQjCCMQI9ggA').card_ids()
+
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

File: Tests/PythonTests/test_deck_info.py [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+    assert deck.delete_card(mage_cards[0].id, 4) is False
+    assert deck.delete_card(druid_cards[0].id, 1) is False
+    deck.show_card_list()
+
+def test_num_card_in_deck():
+    mage_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.MAGE)
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.add_card(mage_cards[0].id, 1)
+
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Tests/PythonTests/test_cards.py  18
- Tests/PythonTests/test_deck_info.py  14
- Tests/PythonTests/test_deck_code.py  49
- Tests/PythonTests/test_card.py  2
- Tests/PythonTests/test_account_info.py  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/RosettaStone/RosettaStone/pullRequest?prid=4593740&bid=15422942)

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Tests/PythonTests/test_deck_info.py  14
- Tests/PythonTests/test_card.py  2
- Tests/PythonTests/test_account_info.py  8
- Tests/PythonTests/test_deck_code.py  49
- Tests/PythonTests/test_cards.py  18
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/RosettaStone/RosettaStone/pullRequest?prid=4593740&bid=15422942)

File: Tests/PythonTests/test_deck_info.py [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+def test_constructors():
+    deck1 = pyRosetta.DeckInfo()
+
+    assert deck1.name() == 'Empty'
+    assert deck1.deck_class() == pyRosetta.CardClass.INVALID
+    assert deck1.num_of_cards() == 0
+
+    deck2 = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+
```
Comment (utilForever) [inline Tests/PythonTests/test_deck_info.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Tests/PythonTests/test_account_info.py  8
- Tests/PythonTests/test_cards.py  18
- Tests/PythonTests/test_card.py  2
- Tests/PythonTests/test_deck_info.py  14
- Tests/PythonTests/test_deck_code.py  49
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/RosettaStone/RosettaStone/pullRequest?prid=4593740&bid=15422942)

File: Builds/setup.py.in [hunk 20]
```diff
@@ -38,7 +38,7 @@ def build_extension(self, ext):
             self.get_ext_fullpath(ext.name)))
         cmake_args = ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
                       '-DPYTHON_EXECUTABLE=' + sys.executable,
-                      '-DBUILD_FROM_PIP=ON']
+                      '-DBUILD_FROM_PIP=ON']
```
Comment (utilForever) [inline Builds/setup.py.in]: Remove whitespace.

File: Extensions/RosettaPython/Sources/Python/Cards/Card.cpp [hunk 21]
```diff
@@ -52,5 +54,43 @@ void AddCard(pybind11::module& m)
...
+            pybind11::arg("player"), pybind11::arg("target"))
+        .def(
+            "get_valid_targets", &Card::GetValidPlayTargets,
+            R"pbdoc(Gets the valid play targets. This method defaults to targeting in the context of spells/hero powers.
+
+			A list of valid play targets.
+
+			Parameters
+			----------
+			- player : The player of the source.)pbdoc",
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Cards/Card.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp [hunk 22]
```diff
@@ -0,0 +1,280 @@
...
+                                        parent);
+    }
+
+    static handle cast(const std::vector<Card*>* src,
+                       return_value_policy policy, handle parent)
+    {
+        return cast(*src, policy, parent);
+    }
+};
+}  // namespace pybind11::detail
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp]: What is this? 💭

File: Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp [hunk 23]
```diff
@@ -0,0 +1,280 @@
...
+                    R"pbdoc(Returns a list of all standard cards.)pbdoc")
+        .def_static("all_wild_cards", &CardsWrapper::GetAllWildCards,
+                    R"pbdoc(Returns a list of all wild cards.)pbdoc")
+        .def_static("find_card_by_id", &CardsWrapper::FindCardByID,
+                    R"pbdoc(Returns a card that matches id.
+
+                    Parameters
+                    ----------
+                    id : The ID of the card.)pbdoc",
+					pybind11::return_value_policy::reference,
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp [hunk 24]
```diff
@@ -0,0 +1,280 @@
...
+            max_val : The maximum cost value of the card.)pbdoc",
+            pybind11::arg("min_val"), pybind11::arg("max_val"))
+        .def_static(
+            "find_card_by_health", &CardsWrapper::FindCardByHealth,
+            R"pbdoc(Returns a list of cards whose health is between min_val and max_val.
+
+			Parameters
+			----------
+			min_val : The minimum health value of the card.
+			max_val : The maximum health value of the card.)pbdoc",
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Extensions/RosettaPython/Sources/Python/Enums/ActionEnums.cpp [hunk 25]
```diff
@@ -0,0 +1,45 @@
...
+               ActionType::CHOOSE_MINION_PUT_LOCATION)
+        .value("CHOOSE_TARGET", ActionType::CHOOSE_TARGET)
+        .value("CHOOSE_ONE", ActionType::CHOOSE_ONE);
+
+    m.def("GetMainOpString", GetMainOpString,
+          R"pbdoc(Returns the string of the main operation type.
+
+		  Parameters
+		  ----------
+		  - op : The string of the main operation type.)pbdoc",
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Enums/ActionEnums.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Extensions/RosettaPython/Sources/Python/Loaders/PowerLoader.cpp [hunk 26]
```diff
@@ -0,0 +1,25 @@
...
+{
+    pybind11::class_<PowerLoader>(
+        m, "PowerLoader",
+        R"pbdoc(This class loads power data from card data generators.)pbdoc")
+        .def_static("load", PowerLoader::Load,
+             R"pbdoc(Loads power data from card data generators.
+
+             Parameters
+             ----------
+			 cards : Data storage to store added cards with power.)pbdoc", pybind11::arg("cards"));
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Loaders/PowerLoader.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp [hunk 27]
```diff
@@ -0,0 +1,81 @@
...
+        .def_static(
+            "req_dragon_target", &TargetingPredicates::ReqDragonTarget,
+            R"pbdoc(Predicate wrapper for checking the target requires that it has Race::DRAGON.)pbdoc")
+        .def_static(
+            "req_target_with_race", &TargetingPredicates::ReqTargetWithRace,
+            R"pbdoc(Predicate wrapper for checking the target requires that it has race.
+
+            Parameters
+            ----------
+			race : The value of race.)pbdoc")
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp [hunk 28]
```diff
@@ -0,0 +1,81 @@
...
+        .def_static(
+            "req_undamaged_target", &TargetingPredicates::ReqUndamagedTarget,
+            R"pbdoc(Predicate wrapper for checking the target requires that it is undamabed.)pbdoc")
+        .def_static(
+            "req_target_max_attack", &TargetingPredicates::ReqTargetMaxAttack,
+            R"pbdoc(Predicate wrapper for checking the target requires that the maximum attack is value.
+
+            Parameters
+            ----------
+			value : The value of maximum attack.)pbdoc")
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp [hunk 29]
```diff
@@ -0,0 +1,81 @@
...
+        .def_static(
+            "req_target_min_attack", &TargetingPredicates::ReqTargetMinAttack,
+            R"pbdoc(Predicate wrapper for checking the target requires that the minimum attack is value.
+
+            Parameters
+			----------
+			value : The value of minimum attack.)pbdoc")
```
Comment (utilForever) [inline Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp]: Rearrange code style. (I think this indent is incorrect.)

File: Tests/PythonTests/test_account_info.py [hunk 30]
```diff
@@ -0,0 +1,44 @@
...
+    assert player.num_of_deck() == 2
+    player.show_deck_list()
+
+    player.delete_deck(0)
+    assert player.num_of_deck() == 1
+    assert player.deck(0).name() == 'deck3'
+
+    assert player.num_of_deck() == 1
```
Comment (utilForever) [inline Tests/PythonTests/test_account_info.py]: I think this test code is unnecessary.

File: Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp (hunk 22 above)
Comment (JYPark09) [reply Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp]: To resolve "Invalid map<K, V> key" error, I wrote above code.
Without this code, python may be destroy Card instance.

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Tests/PythonTests/test_cards.py  18
- Tests/PythonTests/test_deck_info.py  14
- Tests/PythonTests/test_card.py  2
- Tests/PythonTests/test_account_info.py  8
- Tests/PythonTests/test_deck_code.py  49
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/RosettaStone/RosettaStone/pullRequest?prid=4593740&bid=15422942)

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Tests/PythonTests/test_deck_info.py  14
- Tests/PythonTests/test_deck_code.py  49
- Tests/PythonTests/test_card.py  2
- Tests/PythonTests/test_cards.py  18
- Tests/PythonTests/test_account_info.py  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/RosettaStone/RosettaStone/pullRequest?prid=4593740&bid=15422942)

File: Tests/PythonTests/test_account_info.py (hunk 30 above)
Comment (JYPark09) [reply Tests/PythonTests/test_account_info.py]: I removed 44 line and also remove same test code in ```AccountInfoTests.cpp```

File: Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp (hunk 29 above)
Comment (JYPark09) [reply Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp]: I fixed it.

File: Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp (hunk 28 above)
Comment (JYPark09) [reply Extensions/RosettaPython/Sources/Python/Loaders/TargetingPredicates.cpp]: I fixed it.

**Comment by JYPark09** [comment]
I fixed all. Check again please.

**Pull request merged**